scripts/ci/tags.yaml

```diff
@@ -949,9 +949,12 @@ ci_tests_subsys_mpsl:
 
 ci_tests_subsys_nrf_compress:
   files:
-    - nrf/subsys/nrf_compress/
+    - bootloader/mcuboot/
     - nrf/include/nrf_compress/
+    - nrf/subsys/nrf_compress/
     - nrf/tests/subsys/nrf_compress/
+    - zephyr/subsys/dfu/
+    - zephyr/subsys/storage/
```

Comment:
bootloader/mcuboot, zephyr/* are not doing anything with compression

Reply:
Here: https://github.com/nrfconnect/sdk-nrf/tree/main/tests/subsys/nrf_compress/decompression
there are some compression tests and mcuboot_update so both compress and mcuboot is used, thus such depencies.
Those decencies are generated form build system and merged from those 3 tests.

Reply:
the relation is the other way round @nordicjm.It's the compression tests that we want to trigger if mcuboot changed, so no regression leaks when changes in mcuboot breaking compression are to be introduced